src/pipeline.cc

```diff
@@ -689,6 +689,15 @@ class PipelineWorker : public Napi::AsyncWorker {
       baton->channels = image.bands();
       baton->width = image.width();
       baton->height = image.height();
+
+      bool const supportsGifOutput = false;
```

Comment:
Can you point me in the right direction how to detect if `*magick` save is available?

Reply:
Calls to `vips_type_find("VipsOperation", "magicksave")` and `vips_type_find("VipsOperation", "magicksave_buffer")` should return zero when unavailable.

Reply:
Thanks. I will try it.